runtime-sdk/src/runtime.rs

```diff
@@ -19,7 +20,11 @@ pub trait Runtime {
     /// Runtime version.
     const VERSION: version::Version;
 
-    type Modules: AuthHandler + MigrationHandler + MethodRegistrationHandler + BlockHandler;
+    type Modules: AuthHandler
```

Comment:
> Is there a reason these are here and not trait bounds on `Module`. @kostko ?

It could be done either way and we always only use modules here so maybe we can move them (in a subsequent issue/PR of course).